supervisord.conf

```diff
@@ -27,6 +27,17 @@ stdout_logfile = /var/log/mysql.log
 stderr_logfile = /var/log/mysql.log
 autorestart = true
 
+[program:postgresql]
+priority = 1
+command =
+    /usr/pgsql-10/bin/postgres
+        -D /srv/postgres
+        -r /srv/logs/postgres.log
```

Comment:
That will not work. First, `-r` works only in single-user mode. Second, we want postgres to log to stdout/stderr and let supervisor redirect those streams to file (that's what lines below are doing).

Reply:
The parameter has been removed